neutrino.go

```diff
@@ -445,7 +445,7 @@ func (sp *ServerPeer) OnReject(_ *peer.Peer, msg *wire.MsgReject) {
 func (sp *ServerPeer) OnCFHeaders(p *peer.Peer, msg *wire.MsgCFHeaders) {
 	log.Tracef("Got cfheaders message with %d items from %s",
 		len(msg.FilterHashes), p.Addr())
-	sp.server.blockManager.QueueCFHeaders(msg, sp)
+	//sp.server.blockManager.QueueCFHeaders(msg, sp)
```

Comment:
Right, I had to unwire the existing requests for and processing of cfheaders messages because I'm wiring in the new stuff, especially w.r.t. storage. If I try to process the cfheaders through both paths, the store ends up getting messed up.